github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -169,12 +169,13 @@ presubmits:
     annotations:
       fork-per-release: "true"
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
-    always_run: false
+    always_run: true
     optional: true
+    skip_report: false
```

Comment:
i think there is a problem if this is false with the combination of the two above
we had something about it in the past
@qinqon ?

Reply:
I think you are referring to the status reconciler bug here. This is triggered in the following situation: when suddenly a job's value for `optional` changes to `false` (aka making it required), while the release branch configuration still being `optional: true`, the status reconciler tries to forever update the job status, and steadily retriggers the job. But this does not apply here as the optional value is still true.

[See logic here.](https://github.com/kubernetes/test-infra/blob/b1ecafbe3987377c2fc78a97558e8edb04f04588/prow/statusreconciler/controller.go#L332)

@rmohr can you confirm this?

For reference: in such situations it helps changing the forked release branch job config like this:
```
  # please note the added suffix '-release-0.33' on the name and the added context  
  - name: pull-kubevirt-generate-release-0.33
    cluster: ibm-prow-jobs
    branches:
    - release-0.33
    optional: true
    context: pull-kubevirt-generate   

```

Reply:
Thanks Daniel, and Quique,
i think this what Quique also remembered.

Reply:
I would suggest doing the change (adding suffix and pointing to context) upfront so you don't need to remember that later on, when you want to make the job required.

Reply:
You mean I need to add `context: pull-kubevirt-e2e-kind-1.17-sriov` to the release job of this lane?

Reply:
On this file https://github.com/kubevirt/project-infra/blob/master/github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.34.yaml#L440 ?